README.md

```diff
@@ -55,17 +61,365 @@ npm i @xmtp/react-native-sdk
 npx pod-install
 ```
 
+<!--any thoughts on providing more details here?-->
+
 We're working on testing the end-to-end installation and will provide more platform-specific configuration details.
 
 ### Configure for Android
 
 Your app must use Android `minSdkVersion = 22` to work with the `xmtp-react-native` SDK.
 
+<!--any thoughts on providing more details here?-->
```

Comment:
See comment

Reply:
That's the only item that is needed for android specifically.